modules/data-entry/src/main/frontend/src/dataHomepage/SubjectView.jsx

```diff
@@ -0,0 +1,130 @@
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+import React, { useState } from "react";
+import LiveTable from "./LiveTable.jsx";
+
+import QuestionnaireStyle from "../questionnaire/QuestionnaireStyle.jsx";
+
+import {
+  Button,
+  Card,
+  CardContent,
+  CardHeader,
+  CircularProgress,
+  Divider,
+  Tab,
+  Tabs,
+  Typography,
+  withStyles
+} from "@material-ui/core";
+import DeleteButton from "./DeleteButton.jsx";
+import { getEntityIdentifier } from "../themePage/EntityIdentifier.jsx";
+
+function SubjectView(props) {
+  const { classes } = props;
+  const [ activeTab, setActiveTab ] = useState(0);
+  const [ subjectTypes, setSubjectTypes] = useState([])
+  const [ tabsLoading, setTabsLoading ] = useState(null);
+  const hasSubjects = tabsLoading === false && subjectTypes.length > 0;
+
+  // Column configuration for the LiveTables
+  const columns = [
+    {
+      "key": "@name",
+      "label": "Identifier",
+      "format": getEntityIdentifier,
+      "link": "dashboard+path",
+    },
+    {
+      "key": "jcr:created",
+      "label": "Created on",
+      "format": "date:YYYY-MM-DD HH:mm",
+    },
+    {
+      "key": "jcr:createdBy",
+      "label": "Created by",
+      "format": "string",
+    },
+  ]
+  const actions = [
+    DeleteButton
+  ]
+
+  let fetchSubjectTypes = () => {
+    let url = new URL("/query", window.location.origin);
+    url.searchParams.set("query", `SELECT * FROM [lfs:SubjectType] as n order by n.'lfs:defaultOrder'`);
+    // TODO: Handle pagination?
+    url.searchParams.set("limit", 10);
+    url.searchParams.set("offset", 0);
+    return fetch(url)
+      .then(response => response.json())
+      .then(result => {
+        setSubjectTypes(result.rows);
+        setTabsLoading(false);
+      })
+  }
+
+  if (tabsLoading == null) {
+    fetchSubjectTypes();
+    setTabsLoading(true);
+  }
+
+  let path;
+  if (hasSubjects) {
+    path = subjectTypes[activeTab]['@path'];
+  }
+
+  return (
+    <Card>
+      <CardHeader
+        title={
+          <Button className={classes.cardHeaderButton}>
+            Subjects
+          </Button>
+        }
+      />
+      {
+        tabsLoading
+          ? <CircularProgress/>
+          : <Tabs value={activeTab} onChange={(event, value) => setActiveTab(value)}>
+              {subjectTypes.map((subject, index) => {
+                return <Tab label={subject['label'] || subject['@name']} key={"subject-" + index}/>;
+              })}
+            </Tabs>
+      }
+      <Divider />
+      <CardContent>
+      {
+        hasSubjects
+          ? <LiveTable
+              columns={columns}
+              customUrl={'/Subjects.paginate?fieldname=type&fieldvalue='+ encodeURIComponent(subjectTypes[activeTab]["jcr:uuid"])}
+              defaultLimit={10}
+              filters
```

Comment:
No filters here, they don't work as expected on subjects/

```suggestion
              filters
```